addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -141,11 +141,10 @@ FormRenderer.include({
      * @override
      */
     _renderNode(node) {
-        if (
-            !this._isFromFormViewDialog &&
-            node.tag === 'div' &&
-            node.attrs.class === 'oe_chatter'
-        ) {
+        if (node.tag === 'div' && node.attrs.class === 'oe_chatter') {
+            if (this._isFromFormViewDialog) {
+                return;
```

Comment:
Would return a `<div/>` so that it's consistent with `_renderNode` documentation. It's weird to return `undefined`.